### PR TITLE
[GEOS-9258] Promote status-monitoring module from Community to Core (GSIP-175)

### PR DESCRIPTION
**Pull request by nmco**
## Description

Moves the community status-monitoring module to extension status:
- moves and updates the module source code from community to extension folder
- moves and updates the plugin ZIP descriptor
- updates ``web app`` module dependency  ``status-monitoring``
- moves and updates the documentation

Associated JIRA issues:
https://osgeo-org.atlassian.net/browse/GEOS-9258

Associated _GeoServer Improvement Proposal_ (GIP):
https://github.com/geoserver/geoserver/wiki/GSIP-175

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)

The following are required only for core and extension modules, while they are welcomed, but not required, for community modules:
- [x] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [x] New unit tests have been added covering the changes
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by travis-ci after opening this PR)
- [x] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates

Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.

### CODE REVIEW

**Comment by nmco** [comment]
Local build using:
- mvn clean install -DskipTests -T4 -Prelease,communityRelease -fn
- mvn assembly:attached
- mvn clean install -DskipTests -T4 -Pstatus-monitoring

Build was successful (see bellow), ZIP plugin file was correctly created and GeoServer WAR contains the plugin when build whit the ``status-monitoring`` profile:

 ```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] GeoServer .......................................... SUCCESS [  2.607 s]
[INFO] Core Platform Module ............................... SUCCESS [  2.825 s]
[INFO] Open Web Service Module ............................ SUCCESS [  1.533 s]
[INFO] Main Module ........................................ SUCCESS [  7.070 s]
[INFO] GeoServer Security Modules ......................... SUCCESS [  0.048 s]
[INFO] GeoServer Security Tests Module .................... SUCCESS [  1.444 s]
[INFO] GeoServer JDBC Security Module ..................... SUCCESS [  0.827 s]
[INFO] GeoServer LDAP Security Module ..................... SUCCESS [  0.918 s]
[INFO] Web Coverage Service Module ........................ SUCCESS [  0.875 s]
[INFO] Web Coverage Service 1.0 Module .................... SUCCESS [  1.225 s]
[INFO] Web Coverage Service 1.1 Module .................... SUCCESS [  1.668 s]
[INFO] Web Coverage Service 2.0 Module .................... SUCCESS [  1.773 s]
[INFO] Web Feature Service Module ......................... SUCCESS [  4.444 s]
[INFO] Web Map Service Module ............................. SUCCESS [  5.034 s]
[INFO] KML support for GeoServer .......................... SUCCESS [  1.260 s]
[INFO] gs-rest ............................................ SUCCESS [  1.577 s]
[INFO] GeoWebCache (GWC) Module ........................... SUCCESS [  1.708 s]
[INFO] gs-restconfig ...................................... SUCCESS [  2.735 s]
[INFO] gs-restconfig-wcs .................................. SUCCESS [  0.554 s]
[INFO] gs-restconfig-wfs .................................. SUCCESS [  0.501 s]
[INFO] gs-restconfig-wms .................................. SUCCESS [  0.443 s]
[INFO] GeoServer Web Modules .............................. SUCCESS [  0.036 s]
[INFO] Core UI Module ..................................... SUCCESS [  5.266 s]
[INFO] WMS UI Module ...................................... SUCCESS [  1.122 s]
[INFO] GWC UI Module ...................................... SUCCESS [  0.980 s]
[INFO] WFS UI Module ...................................... SUCCESS [  0.731 s]
[INFO] Demos Module ....................................... SUCCESS [  0.795 s]
[INFO] WCS UI Module ...................................... SUCCESS [  0.513 s]
[INFO] Security UI Modules ................................ SUCCESS [  0.036 s]
[INFO] Security UI Core Module ............................ SUCCESS [  1.819 s]
[INFO] Security UI JDBC Module ............................ SUCCESS [  0.664 s]
[INFO] Security UI LDAP Module ............................ SUCCESS [  0.619 s]
[INFO] REST UI Module ..................................... SUCCESS [  0.537 s]
[INFO] GeoServer Web Application .......................... SUCCESS [  1.531 s]
[INFO] Community Space .................................... SUCCESS [  0.053 s]
[INFO] Authentication by unique url key ................... SUCCESS [  0.634 s]
[INFO] WorldWind Format Module ............................ SUCCESS [ 12.403 s]
[INFO] GeoServer JDBC KV Catalog .......................... SUCCESS [  0.855 s]
[INFO] GeoServer Extensions ............................... SUCCESS [  0.056 s]
[INFO] Web Processing Service parent ...................... SUCCESS [  0.025 s]
[INFO] Web Processing Service Module ...................... SUCCESS [  2.497 s]
[INFO] GeoServer Scripting Extensions ..................... SUCCESS [  0.024 s]
[INFO] Core Scripting Extension ........................... SUCCESS [  0.646 s]
[INFO] Web UI Scripting Extension ......................... SUCCESS [  0.465 s]
[INFO] BeanShell Scripting Extension ...................... SUCCESS [  1.812 s]
[INFO] H2 DataStore Extension ............................. SUCCESS [  0.579 s]
[INFO] Python Scripting Extension ......................... SUCCESS [ 28.099 s]
[INFO] WMS-EO ............................................. SUCCESS [  0.639 s]
[INFO] ColorMap ........................................... SUCCESS [  0.465 s]
[INFO] GeoServer mbtiles Extension ........................ SUCCESS [  0.491 s]
[INFO] GeoServer GeoPackage Extension ..................... SUCCESS [  0.549 s]
[INFO] PGRaster DataStore Extension for automatic configuration SUCCESS [  1.247 s]
[INFO] Dynamic WMS dimension value defaults ............... SUCCESS [  0.554 s]
[INFO] GPX PPIO for WPS ................................... SUCCESS [  0.511 s]
[INFO] KML PPIO for WPS ................................... SUCCESS [  0.409 s]
[INFO] DownloadProcess .................................... SUCCESS [  4.672 s]
[INFO] GeoServer JMS based clustering modules ............. SUCCESS [  1.276 s]
[INFO] ActiveMQ ........................................... SUCCESS [ 39.901 s]
[INFO] GeoServer JMS clustering Commons module ............ SUCCESS [  0.222 s]
[INFO] GeoServer JMS clustering module .................... SUCCESS [  3.582 s]
[INFO] GeoServer Hazelcast Clustering Module .............. SUCCESS [  9.145 s]
[INFO] Solr Module ........................................ SUCCESS [  0.555 s]
[INFO] Parameters Extractor ............................... SUCCESS [  0.738 s]
[INFO] GeoServer Data Importer ............................ SUCCESS [  0.045 s]
[INFO] Importer Core Module ............................... SUCCESS [  0.947 s]
[INFO] Web Processing Service - Remote processes invocation SUCCESS [ 31.727 s]
[INFO] WPS cluster on jdbc datastores ..................... SUCCESS [  0.476 s]
[INFO] ncWMS compatibility extension ...................... SUCCESS [  0.465 s]
[INFO] GWC Distributed Caching Plugin ..................... SUCCESS [  1.320 s]
[INFO] GeoWebCache S3 ..................................... SUCCESS [  8.848 s]
[INFO] OGR parent ......................................... SUCCESS [  0.027 s]
[INFO] OGR CORE ........................................... SUCCESS [  0.405 s]
[INFO] GDAL translate parent .............................. SUCCESS [  0.027 s]
[INFO] GDAL translate WCS ................................. SUCCESS [  0.724 s]
[INFO] GDAL translate WPS ................................. SUCCESS [  0.376 s]
[INFO] GeoServer JDBC Configuration Resource Store ........ SUCCESS [  0.427 s]
[INFO] GeoWebCache SQLite ................................. SUCCESS [  1.750 s]
[INFO] GeoServer Config Backup And Restore ................ SUCCESS [  0.137 s]
[INFO] GeoServer Config Backup And Restore Core Module .... SUCCESS [  0.902 s]
[INFO] GeoServer Config Backup And Restore REST Module .... SUCCESS [  0.550 s]
[INFO] GeoServer Config Backup And Restore Extensions ..... SUCCESS [  0.297 s]
[INFO] GeoServer Config Backup And Restore Web UI Module .. SUCCESS [  0.412 s]
[INFO] WMTS Multi Dimensional ............................. SUCCESS [  0.689 s]
[INFO] GeoServer CSS Styling .............................. SUCCESS [  0.426 s]
[INFO] GeoServer Mapbox Styling ........................... SUCCESS [  0.371 s]
[INFO] WMTS Style editing extension ....................... SUCCESS [  0.454 s]
[INFO] GeoServer Security Community Modules ............... SUCCESS [  0.058 s]
[INFO] GeoServer OAuth2 Connect Security Module - Core .... SUCCESS [  7.166 s]
[INFO] GeoServer OAuth2 Connect Security Module - Google Token Service SUCCESS [  0.484 s]
[INFO] GeoServer OAuth2 Connect Security Module - GitHub Token Service SUCCESS [  0.551 s]
[INFO] GeoServer OAuth2 Connect Security Module - GeoNode Token Service SUCCESS [  0.545 s]
[INFO] GeoServer OAuth2 Connect Security Module - OpenId Connect SUCCESS [  0.406 s]
[INFO] GeoServer Keycloak Security Plugin ................. SUCCESS [ 12.906 s]
[INFO] GeoServer OneLogin Module .......................... SUCCESS [ 16.940 s]
[INFO] gs-notification-common ............................. SUCCESS [  4.388 s]
[INFO] GeoServer notification event system extension for GeoNode SUCCESS [  0.348 s]
[INFO] OWS Dispatcher Simulate Plugin ..................... SUCCESS [  0.511 s]
[INFO] JDBC Metrics Plugin ................................ SUCCESS [  0.249 s]
[INFO] OpenSearch EO parent ............................... SUCCESS [  0.082 s]
[INFO] OpenSearch for EO .................................. SUCCESS [  1.094 s]
[INFO] OpenSearch for EO web UI ........................... SUCCESS [  0.471 s]
[INFO] OpenSearch for EO REST configuration API ........... SUCCESS [  0.652 s]
[INFO] NSG Profiles ....................................... SUCCESS [  0.148 s]
[INFO] NSG WMTS 1.0 Profile ............................... SUCCESS [  0.235 s]
[INFO] NSG WFS 2.0 Profile ................................ SUCCESS [  0.761 s]
[INFO] NetCDF Coverage format ............................. SUCCESS [  0.658 s]
[INFO] GRIB Coverage Format ............................... SUCCESS [  0.901 s]
[INFO] WCS NetCDF output Module ........................... SUCCESS [  0.683 s]
[INFO] NetCDF GHRSST encoder .............................. SUCCESS [  0.493 s]
[INFO] Vector Tiles ....................................... SUCCESS [  0.836 s]
[INFO] Application Schema Support ......................... SUCCESS [  0.024 s]
[INFO] Application Schema Integration Test ................ SUCCESS [  1.008 s]
[INFO] WFS 3 prototype .................................... SUCCESS [ 20.235 s]
[INFO] MapML Support for GeoServer ........................ SUCCESS [  9.813 s]
[INFO] S3 Geotiff Support ................................. SUCCESS [  0.391 s]
[INFO] GeoServer Resource Browser ......................... SUCCESS [  0.863 s]
[INFO] GeoServer Task Manager ............................. SUCCESS [  7.491 s]
[INFO] GeoServer Task Manager Core ........................ SUCCESS [ 32.842 s]
[INFO] GeoServer Task Manager S3 .......................... SUCCESS [  2.126 s]
[INFO] GeoServer Quality of Service GetCapabilities module  SUCCESS [  2.476 s]
[INFO] OGR datastore panel ................................ SUCCESS [  0.353 s]
[INFO] GeoServer Community Release Module ................. SUCCESS [  0.289 s]
[INFO] GeoServer notification event system ................ SUCCESS [ 49.692 s]
[INFO] Sample DataAccess Integration Test ................. SUCCESS [  0.520 s]
[INFO] ArcSDE DataStore Extension ......................... SUCCESS [  0.641 s]
[INFO] SQL Server DataStore Extension ..................... SUCCESS [  0.034 s]
[INFO] Oracle DataStore Extension ......................... SUCCESS [  0.040 s]
[INFO] MySQL DataStore Extension .......................... SUCCESS [  0.045 s]
[INFO] DB2 DataStore Extension ............................ SUCCESS [  0.039 s]
[INFO] ImageMap Output Format ............................. SUCCESS [  0.787 s]
[INFO] ImageI/O-Ext GDAL Coverage Extension ............... SUCCESS [  0.269 s]
[INFO] JP2K Coverage Extension ............................ SUCCESS [  0.067 s]
[INFO] OGR WFS ............................................ SUCCESS [  0.501 s]
[INFO] OGR WPS ............................................ SUCCESS [  0.384 s]
[INFO] Excel Output Format ................................ SUCCESS [  0.581 s]
[INFO] Chart external graphics support .................... SUCCESS [  0.042 s]
[INFO] Feature Generalization Extension ................... SUCCESS [  0.283 s]
[INFO] Image Mosaic JDBC Extension ........................ SUCCESS [  0.043 s]
[INFO] OWS request flow controller ........................ SUCCESS [  0.778 s]
[INFO] Web process status sharing with Hazelcast .......... SUCCESS [  0.444 s]
[INFO] Web Processing Service GUI ......................... SUCCESS [  0.719 s]
[INFO] GeoServer Layer Querying filter functions .......... SUCCESS [  0.535 s]
[INFO] Teradata DataStore Extension ....................... SUCCESS [  0.353 s]
[INFO] GeoServer Monitor Extensions ....................... SUCCESS [  0.056 s]
[INFO] Core Monitor Extension ............................. SUCCESS [  0.776 s]
[INFO] GeoServer Security Extension Modules ............... SUCCESS [  0.024 s]
[INFO] GeoServer CAS Security Module ...................... SUCCESS [  0.404 s]
[INFO] GeoServer Security Extension Web Modules ........... SUCCESS [  0.122 s]
[INFO] GeoServer CAS Security Web Module .................. SUCCESS [  0.193 s]
[INFO] XLST based WFS output format ....................... SUCCESS [  0.648 s]
[INFO] GeoServer INSPIRE Extensions ....................... SUCCESS [  0.580 s]
[INFO] Catalog Services for the Web parent ................ SUCCESS [  0.044 s]
[INFO] Catalog Services for the Web interfaces ............ SUCCESS [  1.265 s]
[INFO] Catalog Services for the Web - Simple CatalogStore implementation SUCCESS [  0.580 s]
[INFO] Catalog Services for the Web core module ........... SUCCESS [  1.296 s]
[INFO] CSW UI Module ...................................... SUCCESS [  0.340 s]
[INFO] DXF WFS output format and WPS PPIO ................. SUCCESS [  0.027 s]
[INFO] DXF WFS output format .............................. SUCCESS [  0.606 s]
[INFO] DXF WPS PPIO ....................................... SUCCESS [  0.349 s]
[INFO] Printing Module .................................... SUCCESS [  0.793 s]
[INFO] GeoServer WCS 2.0 Earth Observation Extensions ..... SUCCESS [  0.018 s]
[INFO] Web Coverage Service 2.0 Earth Observation extensions SUCCESS [  0.787 s]
[INFO] Web Coverage Service 2.0 Earth Observation GUI extension SUCCESS [  0.349 s]
[INFO] GeoServer libjpeg-turbo Module ..................... SUCCESS [  0.628 s]
[INFO] Importer REST Api Module-ng ........................ SUCCESS [  0.641 s]
[INFO] Importer Berkley DB Persistence Module ............. SUCCESS [  0.390 s]
[INFO] Importer Web UI Module ............................. SUCCESS [  0.635 s]
[INFO] Ysld GeoServer Plugin .............................. SUCCESS [  0.449 s]
[INFO] MongoDB Data Store ................................. SUCCESS [  0.054 s]
[INFO] REST SLD service ................................... SUCCESS [  1.058 s]
[INFO] GeoFence security integration ...................... SUCCESS [  0.775 s]
[INFO] GeoFence Server .................................... SUCCESS [  0.729 s]
[INFO] Status Monitoring Extension ........................ SUCCESS [  0.808 s]
[INFO] GeoServer Release Module ........................... SUCCESS [  0.264 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:05 min (Wall Clock)
[INFO] Finished at: 2019-06-18T18:42:10+01:00
[INFO] Final Memory: 557M/2151M
[INFO] ------------------------------------------------------------------------
```

**Comment by nmco** [comment]
Valid error prone failure:
!! [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project gs-status-monitoring: Compilation failure
[ERROR] /home/travis/build/geoserver/geoserver/src/extension/status-monitoring/src/main/java/org/geoserver/status/monitoring/collector/MetricValue.java:[115,33] [FormatString] illegal format conversion: 'java.lang.Object' cannot be formatted using '%f'
[ERROR]     (see https://errorprone.info/bugpattern/FormatString)
[ERROR] -> [Help 1]

**Comment by nmco** [comment]
@fernandor777, thx a lot for the help whit the module promotion! Question, for the internationalization would it no be better to add the needed properties to the usual ``GeoServerApplication.properties``, this should make things clearer and easier for translators, what do you think?

**Comment by nmco** [comment]
There is a new build failure, related with PMD:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-pmd-plugin:3.11.0:check (default) on
project gs-status-monitoring: You have 4 PMD violations. For more details see:
/home/travis/build/geoserver/geoserver/src/extension/status-monitoring/target/pmd.xml -> [Help 1]
```

**Comment by fernandor777** [comment]
Thanks @nmco , added GeoServerApplication.properties default values and removed unused fields.

**Comment by fernandor777** [comment]
A proposal for integrate system-status module into Geoserver core:

System status logic classes:
Rename package ```org.geoserver.status.monitoring.collector``` to ```org.geoserger.systemstatus```
Put it in ```gs-main`` module.

Web classes:
Rename package ```org.geoserver.status.monitoring.web``` to ```org.geoserver.web.systemstatus```
Put it in ```gs-web-core``` module.

Rest classes:
rename package ```org.geoserver.status.monitoring.rest``` to ```org.geoserver.rest.systemstatus```
Put it in ```gs-restconfig``` module.

What do you think @nmco ?

**Comment by nmco** [comment]
+1 @fernandor777

**Comment by nmco** [comment]
Summarizing the discussion on the ML, and direction we went:
- the module will be renamed to ``system-status``
- in 2.16.x the module will land on core directly
- for 2.15.x it will be contributed as an extension

So this PR contribute the module to core and another PR will contribute it as an extension.

File: src/web/app/pom.xml [hunk 1]
```diff
@@ -1594,7 +1594,7 @@
       <id>status-monitoring</id>
       <dependencies>
         <dependency>
-          <groupId>org.geoserver.community</groupId>
+          <groupId>org.geoserver.extension</groupId>
```
Comment (nmco) [inline src/web/app/pom.xml]: If the module comes to core this shoudl go away.

File: doc/en/user/source/configuration/status.rst [hunk 2]
```diff
@@ -100,3 +100,203 @@ Module Status Field Descriptions
    Module Status popup
      
 
+System Status
+-------------
+
+System Status adds some extra information about the system in the GeoServer status page in a tab named ``Monitoring``
```
Comment (nmco) [inline doc/en/user/source/configuration/status.rst]: That tab shoudl be renamed ``System Status``, in the doc and in the code.

File: doc/en/user/source/configuration/status.rst [hunk 3]
```diff
@@ -100,3 +100,203 @@ Module Status Field Descriptions
...
+If the ``Monitoring`` tab is not present, it means that the plugin was not installed correctly. The ``Monitoring`` tab content will be refreshed automatically every second.  
+
+REST interface
+^^^^^^^^^^^^^^
+
+It is possible to request the available system information (monitoring data) through GeoServer REST API. The supported formats are XML, JSON and HTML. 
+
+The available REST endpoints are: ::
+
+    /geoserver/rest/about/monitoring
```
Comment (nmco) [inline doc/en/user/source/configuration/status.rst]: Same here, is not monitoring anymore but ``system-status``.

File: doc/en/user/source/configuration/status.rst [hunk 4]
```diff
@@ -73,8 +73,8 @@ The modules tab provides a summary of the status of all installed modules in the
    
    Module Status
```
Comment (nmco) [inline doc/en/user/source/configuration/status.rst]: Where will this documentation page be? Can you show ti using GeoSevrer doc?

File: src/main/src/main/java/org/geoserver/systemstatus/BaseSystemInfoCollector.java [hunk 5]
```diff
@@ -2,7 +2,7 @@
  * This code is licensed under the GPL 2.0 license, available at the root
  * application directory.
  */
-package org.geoserver.status.monitoring.collector;
+package org.geoserver.systemstatus;
```
Comment (nmco) [inline src/main/src/main/java/org/geoserver/systemstatus/BaseSystemInfoCollector.java]: I read you proposition to quickly, IMHO it should be ``org.geoserver.system.status`` instead of ``org.geoserver.systemstatus``.

File: src/main/src/main/java/org/geoserver/systemstatus/MetricValue.java [hunk 6]
```diff
@@ -112,7 +105,11 @@ public String getValueUnit() {
             return BaseSystemInfoCollector.DEFAULT_VALUE;
         }
         if (value instanceof Double || value instanceof Float) {
-            return String.format("%.2f %s", value, unit);
+            final Number numberValue = (Number) value;
+            return String.format(
```
Comment (nmco) [inline src/main/src/main/java/org/geoserver/systemstatus/MetricValue.java]: Why this change?

File: doc/en/user/source/configuration/status.rst (hunk 4 above)
Comment (nmco) [reply doc/en/user/source/configuration/status.rst]: Answering myself:
https://docs.geoserver.org/stable/en/user/configuration/index.html

... looks like a good location, that say, I would update the introduction text sayign it also describes how to view the GeoServe instance status.

File: src/web/core/src/test/java/org/geoserver/web/systemstatus/SystemStatusMonitorPanelTest.java [hunk 7]
```diff
@@ -29,15 +28,15 @@ public void setupTests() {
     @Test
     public void testLoad() throws Exception {
         tester.assertRenderedPage(StatusPage.class);
-        tester.clickLink("tabs:tabs-container:tabs:2:link", true);
+        tester.clickLink("tabs:tabs-container:tabs:3:link", true);
```
Comment (nmco) [inline src/web/core/src/test/java/org/geoserver/web/systemstatus/SystemStatusMonitorPanelTest.java]: Why this change?

File: src/web/app/pom.xml (hunk 1 above)
Comment (fernandor777) [reply src/web/app/pom.xml]: Removed, thanks.

File: doc/en/user/source/configuration/status.rst (hunk 2 above)
Comment (fernandor777) [reply doc/en/user/source/configuration/status.rst]: Renamed.

File: doc/en/user/source/configuration/status.rst (hunk 3 above)
Comment (fernandor777) [reply doc/en/user/source/configuration/status.rst]: Renamed.

File: src/main/src/main/java/org/geoserver/systemstatus/BaseSystemInfoCollector.java (hunk 5 above)
Comment (fernandor777) [reply src/main/src/main/java/org/geoserver/systemstatus/BaseSystemInfoCollector.java]: Refactored.

File: src/web/core/src/test/java/org/geoserver/web/systemstatus/SystemStatusMonitorPanelTest.java (hunk 7 above)
Comment (fernandor777) [reply src/web/core/src/test/java/org/geoserver/web/systemstatus/SystemStatusMonitorPanelTest.java]: Test not working due to extra injected tab on testing for web/core

File: src/main/src/main/java/org/geoserver/systemstatus/MetricValue.java (hunk 6 above)
Comment (fernandor777) [reply src/main/src/main/java/org/geoserver/systemstatus/MetricValue.java]: As far I remember this was due to automatic error prone maven checking warning.

**Comment by aaime** [comment]
The GSIP is approved, please resolve conflicts (how in the world did they get there to start with, not sure... maybe misformatting...)

**Comment by aaime** [comment]
The branch is not cleanly rebased against master, see all the commits

**Comment by fernandor777** [comment]
Checking this...

**Comment by simboss** [comment]
@fernandor777 so?

**Comment by fernandor777** [comment]
Successfully rebased and ready.

**Comment by simboss** [comment]
let's merge then!
